apps/web/src/components/sections/about-us/how-it-works.tsx

```
@@ -0,0 +1,37 @@
+      title: 'Start Transacting',
+      description: 'Send, receive, and manage your assets with ease.',
+    },
+  ];
+
+  return (
+    <section className="py-16">
+      <div className="container mx-auto px-4">
+        <h2 className="text-3xl font-bold text-center mb-12">How KindFi Works</h2>
+        <div className="grid md:grid-cols-3 gap-8">
+          {steps.map((step, index) => (
+            <div key={index} className="text-center">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)